example-java/runner/src/main/kotlin/Runner.java

```diff
@@ -0,0 +1,103 @@
+import arrow.core.Either;
+import io.p8e.ContractManager;
+import io.p8e.contracts.example.HelloWorldContract;
+import io.p8e.exception.P8eError;
+import io.p8e.functional.ContractErrorHandler;
+import io.p8e.functional.ContractEventHandler;
+import io.p8e.proto.ContractSpecs;
+import io.p8e.proto.contract.HelloWorldExample;
+import io.p8e.proxy.Contract;
+import io.provenance.core.extensions.LoggerExtensionsKt;
+import java.util.UUID;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import kotlin.jvm.functions.Function1;
+import org.slf4j.Logger;
+import io.provenance.core.extensions.ProvenanceExtensionsKt;
+
+public class Runner {
+
+    public static void main(String[] args) throws InterruptedException {
+        Logger log = LoggerExtensionsKt.logger(HelloWorldExample.class.getName());
+
+        String p8eUrl = System.getenv("ENV_URL");
+        String key = System.getenv("AFFILIATE_KEY");
+
+        CountDownLatch latch = new CountDownLatch(1);
+        final boolean[] success = {false};
```

Comment:
can this just be a boolean?

Reply:
compiler doesn't like it, I could switch it to AtomicBoolean, looks easier to understand.